catkin_home/src/arm_vision/include/ProcessingCloud.h

```diff
@@ -0,0 +1,419 @@
+#include <IncludersCloud.h>
+
+// hand limits
+// float lower[3] = { 0, 0.23, .1 };
+// float upper[3] = { 360, .68, .9 };
+
+// float lower[3] = { 0, 0.23, .1 };
+// float upper[3] = { 50, .68, .9 };
+
+// GlobalVariables
+int resolution = 100;
+std::vector<float> NormalHand;
+pcl::PointXYZ Centroid;
```

Comment:
Global variables should be avoided, if not possible at least make them const and put them inside a namespace:
https://docs.microsoft.com/en-us/cpp/cpp/namespaces-cpp?view=vs-2019#:~:text=%20Namespaces%20%28C%2B%2B%29%20%201%20using%20directives.%20The,implicit%20global...%204%20See%20also.%20%20More%20